tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -145,6 +149,56 @@ Created Network And Images Persists As Well As Containers Are Discovered With Co
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${alpine}
     Should Be Equal As Integers  ${rc}  0
 
+
+Container on Open Network And Port Forwarding Persist After Reboot
+    [Setup]     NONE
+
+    Set Test Environment Variables
+
+    Log To Console  Create Port Groups For Container network
+    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.add -vswitch vSwitchLAN open-net
+
+    ${createcommand}=  catenate  SEPARATOR=\ \
+    ...  bin/vic-machine-linux create --debug 1 --name=%{VCH-NAME}
+    ...  --target=%{TEST_URL}%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT}
+    ...  --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD}
+    ...  --force=true --bridge-network=bridge --compute-resource=%{TEST_RESOURCE} --no-tlsverify --no-tls
+    ...  --container-network=open-net --container-network-firewall=open-net:open
+
+    ${output}=  Run  ${createcommand}
+
+    Should Contain  ${output}  Installer completed successfully
+    Get Docker Params  ${output}  ${true}
+    Log To Console  Installer completed successfully: %{VCH-NAME}
+
+    # Create a container on the open network
+    ${open-running}=  Launch Container  vch-restart-open-running  open-net
+    ${open-exited}=  Launch Container  vch-restart-open-exited  open-net  ls
+
+    # Create nginx on the open network and check port forwarding
+    Launch Container With Port Forwarding  webserver  10000  10001  open-net
+
+    # Reboot VCH
+    Reboot VM  %{VCH-NAME}
+    Wiat For VCH Initialization  20x  10 seconds
+
+    # Check if the open container is persisted
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} inspect ${open-running} | jq '.[0].State.Status'
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal  ${output}  \"running\"
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} inspect ${open-exited} | jq '.[0].State.Status'
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal  ${output}  \"exited\"
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start ${open-exited}
+    Should Be Equal As Integers  ${rc}  0
+
+    # Check port forwarding after reboot
+    Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  %{VCH-IP}  10000
```

Comment:
These will not and should not succeed. The ports are exposed on the `open-net` port group directly, not via the endpointVM. We will need a second container using `--net=bridge` and port forwarding to confirm that NAT is preserved.

I suggest changing this check to target the `bridge` container I've requested.

Reply:
We should actually have a check that an Open container with port mappings defined does NOT cause NAT rules to be created on the endpointVM.